README.md

```diff
@@ -220,7 +220,11 @@
 
   ```
   yum install ruby
+  yum install ruby-devel
+  yum install zlib-devel
+
   gem install trollop
+  gem install fog
```

Comment:
right, the yum install ruby-devel/zlib-devel were needed when gem install fog was rebuilding nokogiri.